kong/pdk/plugin.lua

```diff
@@ -0,0 +1,20 @@
+--- Plugin related APIs
+--
+-- @module kong.plugin
+
+
+local _plugin = {}
+
+
+function _plugin.get_id(self)
```

Comment:
Should we make a corresponding adjustment to allow this PDK API to be invoked by other languages? Or it will be exposed automatically?

Reply:
It will be done by manually running a script.